src/js/utils.js

```diff
@@ -0,0 +1,43 @@
+export const getMenuTemplate = (name) => {
+  return `
+    <li class="menu-list-item d-flex items-center py-2" data-menu-name="${name}">
+      <span class="w-100 pl-2 menu-name">${name}</span>
+      <button
+        type="button"
+        class="bg-gray-50 text-gray-500 text-sm mr-1 menu-edit-button"
+      >
+        수정
+      </button>
+      <button
+        type="button"
+        class="bg-gray-50 text-gray-500 text-sm menu-remove-button"
+      >
+        삭제
+      </button>
+    </li>
+  `;
+};
+
+export const renderMenusByFunction = (menus, generateFunction) => {
+  let result = "";
+  menus.forEach((el) => (result += generateFunction(el)));
```

Comment:
`reduce` 를 사용해보셔도 좋을 것 같아요!
```suggestion
 // pseudocode 입니다~!
  return menus.reduce((acc, curr) => acc+= generateFunction(curr) , '')
```

Reply:
좋은것 같아요!! 
수정 코드에서는 map으로 통일했었어서 다음 경우에는 reduce로 깔끔하게 조지겠습니다!